backends/generators/Go/inst.go

```diff
@@ -0,0 +1,2637 @@
+// Code generated by go_generator.py; DO NOT EDIT.
+package riscv
+
+import "cmd/internal/obj"
+
+type inst struct {
```

Comment:
Note there is some discussion about changing this structure and the way we split up the encodings.  See [here](https://github.com/riscv/riscv-opcodes/pull/332) and [here](https://go-review.googlesource.com/c/go/+/637940) so this is likely to change in the future.